src/components/core/components/CardKeyValue/CardKeyValue.tsx

```diff
@@ -60,6 +63,6 @@ export default function CardKeyValue(props: Props) {
 
 CardKeyValue.defaultProps = {
   label: undefined,
-  hideDivider: false,
+  hideDivider: 0,
```

Comment:
hideDivider is boolean. please use correct false value

Reply:
Sorry about that.
Let me fix it